embedded-graphics/src/prelude.rs

```diff
@@ -2,7 +2,7 @@
 
 pub use super::drawable::Drawable;
 pub use super::drawable::Pixel;
-pub use super::fonts::Font;
+//pub use super::fonts::Font;
```

Comment:
Will you add this back before you're done? We export stuff like `Primitive` so this makes sense to leave in, no?

Reply:
Yes, this was a temporary change while I was refactoring this code.